src/api/python/wheels/build_wheel.py

```diff
@@ -49,20 +58,21 @@ def get_cvc5_version():
 
     # read CMakeLists.txt to get version number
     version = dict()
-    str_pattern = 'set\(CVC5_(?P<component>MAJOR|MINOR|RELEASE)\s*(?P<version>\d+)\)'
+    str_pattern = 'set\(CVC5_LAST_RELEASE\s*"([^"]+)"\)'
     pattern = re.compile(str_pattern)
-    with open(os.path.join(project_src_path, 'CMakeLists.txt'), 'r') as f:
+    with open(os.path.join(project_src_path, 'cmake', 'version-base.cmake'), 'r') as f:
```

Comment:
I'd suggest to read the whole file in one go (`f.read()`) and apply the regex once to the whole thing. The file will always be small (so there is no memory issue) and the code will be way easier.

Reply:
Good point. I was going for the "minimal diff" approach, but there's no harm in doing some cleanup.